src/QsCompiler/Tests.Compiler/QirTests.fs

```diff
@@ -136,10 +160,18 @@ let ``QIR bools`` () =
 let ``QIR bigints`` () =
     qirTest false "TestBigInts"
 
+[<Fact>]
+let ``QIR controlled partial applications`` () =
```

Comment:
Just for the sake of a decent test coverage it would be nice to also have a test with some adjoint and controlled adjoint applications.